### DISCUSSION TITLE
Suggested Feature: Progress Interval in time instead of %

### DISCUSSION THREAD

**Discussion by Wanderwardr** [berrystephenw/OctoText#35] [Ideas]
I think that the periodic notification feature is an excellent part of the software, but I was wondering if you could add a time interval as an alternative to a percentage.  It would be useful to get periodic picture notifications in order to be able to automate the "checking in" process.  I have heard it suggested to check in on the printer every 2 hours or so, in order to make sure that no printer failure can cause damage.  

I figure it would be useful for people to get an image every n minutes, with no notification sent if the print time is shorter than the notification time.  This would remove the need for tweaking of the setting if printing a significantly smaller part after a long print.

**Comment by berrystephenw** [comment]
Hmmm. I like the idea. I think I could implement a time based notification pretty easily, but there are some asynchronous events that I would have to deal with that make it interesting... for instance a print finish notice could come and some time later a timer event could fire afterwards. I'll have to think on this one. Thanks for the suggestion!
